strings/tips/xeno.txt

```diff
@@ -32,8 +32,8 @@ As a xeno, remember that neurotox injected by defilers, sentinels, panthers, and
 As a xeno, you can use the crawl through vent verb found in the xeno tab to quickly find and use vents in the floor without having to alt-click them. 
 As a xeno, you can crawl through holes in walls made by acid by SpecialClicking (Default: CTRL) on them or click-dragging yourself on them. If you're too big to pass through, you'll damage the wall instead.
 As a xeno, you can climb over window frames by SpecialClicking (Default: CTRL) on them or click-dragging yourself on them.
-As a xeno, do not camp the landing zones prior to the marines building their defenses.
+As a xeno, do camp the landing zones prior to the marines building their defenses.
```

Comment:
I'm iffy on this one

Reply:
i can remove it easy. that line is sorta misleading. maybe just the removal of that line?

Reply:
What
Do not camp wtf fobcamp bad